tests/nested/manual/core20-factory-reset-install-device-hook/install-device

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# install the files the same way the specific device does this - note we don't
+# create the directories since that requires extra permissions we don't have 
+# here, system-files only allows us access to the specific files
+
+# add a udev rule if executed from factory-reset
+# when not in install mode _writable_defaults are present under /run/mnt/data
+# and not /run/mnt/ubuntu-data
+if snapctl system-mode | grep "factory-reset"; then
+    cat << EOF > /run/mnt/ubuntu-data/system-data/_writable_defaults/etc/udev/rules.d/09-my-custom-udev.rules
+# do udev rules things here
```

Comment:
maybe the comment should be a bit more specific about the what is expected. Something like:
```
# this udev rule is *only* written during factory reset and no other files will be created in this mode
```
and similarly in the modprobe and moudles-load.d file something like:
```
# this file is *only* written during install (but not factory-reset) 
```

Reply:
Good point, will update this!